bedrock/firefox/views.py

```diff
@@ -721,7 +704,9 @@ def get_context_data(self, **kwargs):
 
         reason = self.request.GET.get("reason", None)
         manual_update = True if reason == "manual-update" else False
+        outdated = True if reason == "outdated" else False
```

Comment:
This is fine but also repetitive since `reason == "outdated"` returns a boolean. So it's like saying `True if True else False`. Simpler and a bit more Pythonic would be:
```python
outdated = reason == "outdated"
```